src/BugsnagLogger.php

```diff
@@ -26,6 +34,8 @@ class BugsnagLogger extends AbstractLogger
     public function __construct(Client $client)
     {
         $this->client = $client;
+        $logNotifyLevel = $this->client->getConfig()->getLogLevel();
```

Comment:
This seems like a roundabout way to set the log level by default. Loading it from the config prevents users from configuring the logger directly or separately from their initial Bugsnag configuration. In addition, it is potentially confusing for users who are using bugsnag-php without the logger, and may be conflated with [setErrorReportingLevel](https://docs.bugsnag.com/platforms/php/other/configuration-options/#error-reporting-level) for example. A setter would be preferable, with usage like:

```php
$bugsnag = Bugsnag\Client::make('API KEY');
$logger = new Bugsnag\PsrLogger\BugsnagLogger($bugsnag);

// Set minimum log-level for sending a report to Bugsnag
$logger->setLogReportingLevel(\Psr\Log\LogLevel::ERROR);

// Add breadcrumbs for low-severity log messages
$logger->notice("Reticulating splines");

// Log an exception to Bugsnag
$logger->error(new Exception("Invalid configuration at runtime"));
```

The existing constructor could be preserved and we can have new option for log level solely where needed, such as in `config/bugsnag.php` and an env variable for Laravel. The documentation would be something like:

> ### Log reporting level
>
> The log reporting level is the minimum log level (as specified by [`\Psr\Log\LogLevel`](http://www.php-fig.org/psr/psr-3/#psrlogloglevel)) required to generate an error report to send to Bugsnag. All log messages below the selected level are recorded as breadcrumbs on subsequent reports.
>
> You can set the Log reporting level by setting the `BUGSNAG_LOG_REPORTING_LEVEL` environment variable or by setting `log_reporting_level` within `config/bugsnag.php`:
>
> ```php
> "log_reporting_level" => "error"
> ```
>
> **By default, this is set to `notice`**